bigbluebutton-html5/imports/ui/components/user-list/component.jsx

```diff
@@ -5,7 +5,6 @@ import { withRouter } from 'react-router';
 
 import injectWbResizeEvent from '/imports/ui/components/presentation/resize-wrapper/component';
 import styles from './styles';
-import UserListHeader from './user-list-header/component';
```

Comment:
I can't find where we use user-list-header. The only part is the commented out section in this file.
@oswaldoacauan is there a reason why we kept those files?

Reply:
@tylercopeland asked us to remove the title of the user-list. We probably can remove that file since we dont have a header anymore